### ISSUE TITLE
old SSH algos

### ISSUE THREAD

**Issue by calestyo** [dCache/dcache#6499]
Hey.

I have only dCache 6.0.x here right now, so perhaps this is already a non-issue in current versions, but I guess I'd forget to report that until I have some more current one:

At least in 6.0.x the admin interface's SSH alogs are quite outdated:
```
Unable to negotiate with ::1 port 59999: no matching host key type found. Their offer: ssh-rsa
```
That one will fail with SSH >= 8.8, as `ssh-rsa` (the key signing algo, not the key type of the same name) has been removed from the default list.

```
Unable to negotiate with ::1 port 59999: no matching MAC found. Their offer: hmac-md5,hmac-sha1,hmac-sha2-256,hmac-sha2-512,hmac-sha1-96,hmac-md5-96
```
These are all non-`-etm` versions, which are considered vulnerable to certain attacks (well it depends on the other algos in use actually, but at least when non AEAD-alogs are used).

Just close, if it's a non-issue in current versions.

Thanks,
Chris.

**Comment by paulmillar** [comment]
dCache 6.0 is no longer supported.

Could you upgrade to a supported version of dCache and test whether this is still a problem?

Cheers,
Paul.

**Comment by calestyo** [comment]
I could only test next week, when I upgrade our dCache to... 7.2 or 8.0 ...
But I guess that's no longer needed now, is it?

If not, we can close this issue from my side :-)

**Comment by kofemann** [comment]
Let's wait until your confirmation 👀

**Comment by calestyo** [comment]
Confirmed as fixed in at least 7.2.10 (but probably already much earlier).

Thanks,
Chris.
